zipkin-server/src/main/java/zipkin2/server/internal/banner/ZipkinAnsi256Color.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2015-2019 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin2.server.internal.banner;
+
+import org.springframework.boot.ansi.AnsiElement;
+
+public class ZipkinAnsi256Color implements AnsiElement {
```

Comment:
add a TODO to delete on spring boot 2.2 if that's when this is scheduled to be normally possible.

Reply:
Updated in edebc31428c9a3551181aa053ecb008b9dd2666e